src/tools/pub/pubspec.md

```diff
@@ -74,14 +74,19 @@ A pubspec can have the following fields:
 : Optional. Specify where to publish a package.
   [_Learn more._](#publish_to)
 
-Pub ignores all other fields.
+Pub ignores all other fields,
```

Comment:
Really?

Reply:
@jonasfj or @isoos can you confirm?

Reply:
Yes, at the moment pub site ignores other fields. We had discussions to reject pubspecs that have extra fields, but at the time it was decided to continue ignoring instead.

Reply:
Hmmm... That doesn't sound as ironclad as I thought it was. Do you think there's a chance we'll change our minds?

Reply:
> Do you think there's a chance we'll change our minds?

There is always a small chance/risk we'll change it, I would like to, but that's my opinion and I don't see us having the bandwidth anytime soon... On the other hand, do we need to document that other fields are ignored?

ie. if we don't write anything about it, the "feature" is less likely to be abused. IMO, if we were to write anything, we should encourage users introducing custom properties to avoid using generic names (ie. use unique names, such as their package name, or something like that).